### DISCUSSION TITLE
長途駕駛福音：日本電信公司NTT發表車用聊天機器人，陪你聊風景防睡意！

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#439] [科技新知]
「日本電信公司NTT於30日發表一款車用對話AI，會以車窗外可看見的移動風景作為話題，與駕駛人聊天。據稱對於防止長途駕駛時的睡意、或是注意力渙散等都有幫助。」

我爸超需要這種機器人他開車都需要我跟他聊天。如果這種機器人能有效降低疲勞駕駛恍神車禍狀況也是一大福音，實在值得發展。

資料來源：https://www.thenewslens.com/article/167709?_gl=1*47im45*_ga*MTJWNUY1YjJEU1BRaGZueWRLRUN4X1U3bHF0SEhPT1pBa2NyVlk2Szl5SmxMZnJ0UFJpTkxsOVU1T2xOSElGMg..

**Comment by YuHengChen54** [comment]
感覺很酷，但不知道會不會跟駕駛聊得太開心，反而害駕駛不專心哈哈

**Comment by BingRong1217** [comment]
這會對很多要開長途的司機幫助很大

**Comment by limitx0** [comment]
這個AI的對話好流暢，很像是在跟真人對話

**Comment by brian431** [comment]
在描述外面的風景時最好是只聊前面遠方的東西，不然反而會讓駕駛分心轉頭去看旁邊之類的。

**Comment by awehao** [comment]
我覺得這會讓駕駛分心，一直去看外面的東西，是個雙面刃呢

**Comment by Aerofe** [comment]
如果一直以風景來當作話題的話，感覺久了就會膩了，可能需要再多一點功能

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by larry40305** [comment]
發明AI陪你聊天，為了避免聊到分心，所以要再加入判斷專注力的AI。
